test/CavyTester/src/scenarios/taggingTest.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export const key = 'TaggingSingleTest';
+
+export const Screen = () => {
+  return <></>;
+};
+
+export const label = 'adding a tag to an `it` block filters the test';
+export const spec = spec =>
+  spec.describe(key, () => {
+    spec.it(label, async () => {
+      throw new Error('This test should not run!');
+    })
+  }, 'dontRun');
```

Comment:
This test is a little strange, tagging the test so it wont run...., but I cannot think of a better way to test the functionality without setting up another `Tester`
Overall, I think it is better to have the test even if it is a little odd.

Reply:
I know :( now _every_ test is tagged in the `CavyTester` .. 🤔 

Re your comment about, by unit tests, I mean using traditional Jest tests, to unit test our tagging functionality, rather than using Cavy to test Cavy as we're doing in the CavyTester.